### PR TITLE
Using block store cache

### PR DESCRIPTION
**Pull request by ajlopez**
Adding an in-memory block store cache

(use with blockchain.flush = true; to test in some nodes)

### CODE REVIEW

**Comment by donequis** [comment]
What do we expect with this change?

File: rskj-core/src/main/java/org/ethereum/config/DefaultConfig.java [hunk 1]
```diff
@@ -114,11 +114,11 @@ public BlockStore blockStore() {
 
 
         IndexedBlockStore cache = new IndexedBlockStore();
-        cache.init(new HashMap<Long, List<IndexedBlockStore.BlockInfo>>(), new HashMapDB(), null, null);
+        cache.init(new HashMap<>(), new HashMapDB(), null, null);
```
Comment (m-sossich) [inline rskj-core/src/main/java/org/ethereum/config/DefaultConfig.java]: Consider the repercussion of not using a "concurrent" map implementation here.

**Comment by donequis** [comment]
Create PR against ginger

**Comment by aeidelman** [comment]
This pull request is replaced by PR #119 and will not be merged.

**Pull request closed without merging**